demisto_sdk/commands/setup_env/setup_environment.py

```diff
@@ -0,0 +1,406 @@
+
+    Returns:
+        dict: The integration params
+    """
+    # Create the Secret Manager client.
+    client = secretmanager.SecretManagerServiceClient()
+
+    # Build the resource name of the secret version.
+    name = f"projects/{project_id}/secrets/{secret_id}/versions/latest"
+
+    # Access the secret version.
+    try:
+        response = client.access_secret_version(name=name)
+    except google.api_core.exceptions.NotFound:
+        logger.warning("The secret is not found in the secret manager")
+        return {}
+    except google.api_core.exceptions.PermissionDenied:
+        logger.warning(
+            "Insufficient permissions for gcloud. If you have the correct permissions, run `gcloud auth application-default login`"
+        )
+        return {}
+    except Exception:
+        logger.warning(
+            f"Failed to get secret {secret_id} from secret manager, skipping"
+        )
+        return {}
+    # Return the decoded payload.
+    payload = response.payload.data.decode("UTF-8")
+    return json5.loads(payload).get("params")
+
+
+def add_init_file_in_test_data(integration_script: IntegrationScript):
+    if (integration_script.path.parent / "test_data").exists():
+        (integration_script.path.parent / "test_data" / "__init__.py").touch()
+
+
+def configure_dotenv():
+    """
+    This functions configures the .env file located with PYTHONPATH and MYPYPATH
+    """
+    dotenv_path = CONTENT_PATH / ".env"
+    env_vars = dotenv.dotenv_values(dotenv_path)
+    env_vars["PYTHONPATH"] = ":".join([str(path) for path in PYTHONPATH])
+    env_vars["MYPYPATH"] = ":".join([str(path) for path in PYTHONPATH])
+    for key, value in env_vars.items():
+        if not value:
+            continue
+        dotenv.set_key(dotenv_path, key, value)
+
+
+def configure_vscode_settings(
+    ide_folder: Path, integration_script: IntegrationScript, interpreter_path: Path
+):
+    shutil.copy(Path(__file__).parent / "settings.json", ide_folder / "settings.json")
+    with open(ide_folder / "settings.json") as f:
+        settings = json5.load(f)
+
+    settings["python.defaultInterpreterPath"] = str(interpreter_path)
+    settings["python.testing.cwd"] = str(integration_script.path.parent)
+    with open(ide_folder / "settings.json", "w") as f:
+        json.dump(settings, f, indent=4)
+
+
+def configure_vscode_tasks(
+    ide_folder: Path, integration_script: IntegrationScript, test_docker_image: str
+):
+    if integration_script.type == "powershell":
+        logger.debug("Powershell integration, skipping tasks.json")
+        return
+    docker_python_path = []
+    for path in PYTHONPATH:
+        with contextlib.suppress(ValueError):
+            # we can't add paths which is not relative to CONTENT_PATH, and `is_relative_to is not working on python3.8`
+            docker_python_path.append(
+                f"/app/{path.relative_to(CONTENT_PATH.absolute())}"
+            )
+    tasks = {
```

Comment:
let's build `tasks` in a separate method, potentially an inner one, just to keep the rest of the code readable